core/config/config.go

```diff
@@ -49,15 +49,15 @@ var (
 	// This is the default, Chain development configuration.
 	// These options can be updated with build tags.
 	BuildConfig = struct {
-		LoopbackAuth bool `json:"is_loopback_auth"`
-		MockHSM      bool `json:"is_mockhsm"`
-		Reset        bool `json:"is_reset"`
-		PlainHTTP    bool `json:"is_plain_http"`
+		LocalhostAuth bool `json:"is_localhost_auth"`
```

Comment:
Do you mind not changing this variable name? Please change as little as possible for now. 🙏